### PR TITLE
refactor(console): add fuzzy tips

### LINKED ISSUE #4476: nacos 1.4.0 dashboard config list can't Fuzzy query
**Describe the bug**
image

image

**Expected behavior**

Fuzzy query  by Data ID  , Group 

**Acutally behavior**

the list is empty 

**How to Reproduce**

**Desktop (please complete the following information):**
 - OS: [e.g. Centos]
 - Version [e.g. nacos-server 1.3.1, nacos-client 1.3.1]
 - Module [e.g. naming/config]
 - SDK [e.g. original, spring-cloud-alibaba-nacos, dubbo]

**Additional context**

### PR DESCRIPTION
**Pull request by freshgeek**
add fuzzy tips ,add * in placeholder

Closes #4476

## What is the purpose of the change

add fuzzy tips ,add * in placeholder

## Brief changelog

add fuzzy tips ,add * in placeholder

Fixes #4476

### CODE REVIEW

**Review by horizonzy — COMMENTED** [review]
english version also need add tips

**Comment by freshgeek** [comment]
> english version also need add tips

OK, I'll submit another one

**Comment by freshgeek** [comment]
add fuzzy tips ,add * in placeholder

Closes #4476

## What is the purpose of the change
 
add fuzzy tips ,add * in placeholder for zh-cn and en-us

## Brief changelog
 
add fuzzy tips ,add * in placeholder for zh-cn and en-us

Fixes #4476

File: console-ui/src/locales/zh-CN.js [hunk 1]
```diff
@@ -265,8 +265,8 @@ const I18N_CONF = {
     configurationManagement8: '配置管理',
     queryResults: '查询结果：共查询到',
     articleMeetRequirements: '条满足要求的配置。',
-    fuzzyd: '模糊查询请输入Data ID',
-    fuzzyg: '模糊查询请输入Group',
+    fuzzyd: '模糊查询请输入*Data ID*',
+    fuzzyg: '模糊查询请输入*Group*',
```
Comment (KomachiSion) [inline console-ui/src/locales/zh-CN.js:269]: Is this ambiguous?

Maybe `添加通配符'*'进行模糊查询` will be better?

Comment (freshgeek) [reply console-ui/src/locales/zh-CN.js:269]: feel `添加通配符'*'进行模糊查询`  more suitable for hover than for placeholder

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**